src/release_notes.md

```diff
@@ -0,0 +1,6 @@
+### What's Changed
```

Comment:
This repo doesn't have release notes -- I think these changes can be summarized in this directory. Can open a different PR to make sure all the appropriate directories have a `release_notes.md`.